spyder/config/main.py

```diff
@@ -172,6 +172,11 @@
               'dataframe_format': '.6g',  # No percent sign to avoid problems
                                           # with ConfigParser's interpolation
               'excluded_names': EXCLUDED_NAMES,
+              'exclude_private_preferences': True,
+              'exclude_uppercase_preferences': False,
+              'exclude_capitalized_preferences': False,
+              'exclude_unsupported_preferences': False,
+              'exclude_callables_and_modules_preferences': True,
```

Comment:
The idea around that was to prevent the context menu options changing the options displayed in the preference page following the point `In the edit filters option in preferences the options currently activated by defaullt should still be checked by default.` listed at the end of https://github.com/spyder-ide/ux-improvements/issues/17.  We interpreted that point as having a default filters definition in the config page which doesn't change with the changes done in the context menu.

Is then not necessary to have a default filter options state from the preference page which is not affected for the current context menu checked filters? Or maybe the idea is correct (the preferences page being the default options and only changeable from the preference page) but another approach should be followed @ccordoba12 ?

Reply:
> Is then not necessary to have a default filter options state from the preference page which is not affected for the current context menu checked filters?

That's correct, I don't think it's necessary. That would make this functionality work contrary to how things do in the rest of our plugins.

> Or maybe the idea is correct (the preferences page being the default options and only changeable from the preference page) but another approach should be followed @ccordoba12 ?

Well, I think this would make things inconsistent, which would be confusing for users. For instance, if you check `Exclude capitalized variable` in the Options menu, and then click on the new `Edit filters` context menu entry, users wouldn't see that option checked in Preferences. So, I think we should leave things as they are working right now.

What would be interesting is to apply our filters to each stacked widget in the Variable Explorer (instead to all of them). But since users haven't requested for that functionality, there's no need to implement it just yet.